packages/odyssey-storybook/src/components/odyssey-mui/Tag/Tag.stories.tsx

```diff
@@ -29,83 +29,69 @@ export default {
       control: "text",
       defaultValue: "Starship",
     },
-    label2: {
+    isDisabled: {
+      control: "boolean",
+      defaultValue: false,
+    },
+    onClick: {
       control: "text",
       defaultValue: null,
     },
-    label3: {
+    onDelete: {
```

Comment:
Let's make sure these default values match the default values of the components so our stories match how they will use them.

When these get auto-generated normally, the default values are the default values in our component files, not for stories.

Our default values in Odyssey are both in the component files and in the `theme/components.tsx` file.